modules/configuring-ide.adoc

```diff
@@ -0,0 +1,199 @@
+:_module-type: PROCEDURE
+
+[id="configuring-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring VS Code Server
+
+Incorporating standard database clients into your VS Code Server environment can streamline your workflow, offering a seamless interface for managing databases. You can incorporate different Database Management System (DBMS) clients into your Code Server workbench through built-in VS Code extensions or by creating a custom notebook image using Dockerfile.
+
+=== Incorporating DBMS clients using VS Code extensions
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
```

Comment:
I assume that you also need access to the VS Code software on your local machine?

Reply:
Per chat, this is in-browser.